src/components/molecules/BoothCard/index.tsx

```diff
@@ -60,7 +14,7 @@ export const BoothCard = ({ title, description }: BoothCardProps) => {
         variant="body2"
         sx={{ mb: '24px', maxWidth: '680px', wordBreak: 'break-word', whiteSpace: 'pre-wrap' }}
       >
-        {replaceUrlWithLink(description)}
+        {replaceUrlWithLink(description, { keepFullLength: false })}
```

Comment:
[memo]

https://github.com/GoCon/2023/pull/256#discussion_r1207814734 の変更を入れた場合ここの引数指定は不要になります。

```suggestion
        {replaceUrlWithLink(description)}
```